### PR TITLE
refac(dh): grid area in create calculation

### PR DESCRIPTION
**Pull request by ManBearTM**
This PR moves all of the logic concerning grid areas in create calculations into its own component. I had to skip the cypress test since I just couldn't get it to work (the query would not return the mocked data). I couldn't justify spending any more time on that, so I'm skipping it for now. We will have a talk about it later.

### CODE REVIEW

File: libs/dh/wholesale/feature-calculations/src/lib/create/create.component.html [hunk 1]
```diff
@@ -21,162 +21,111 @@
...
-              {{ t("scheduling.SCHEDULED") }}
-            </watt-filter-chip>
-          </div>
+    <form
+      id="create-calculation"
+      class="grid"
+      novalidate
+      [formGroup]="formGroup"
+      (ngSubmit)="
+        latestPeriodEnd && formGroup.controls.calculationType.value === 'BALANCE_FIXING'
```
Comment (mimse) [inline libs/dh/wholesale/feature-calculations/src/lib/create/create.component.html:30]: Suggestion: import and expose CalculationType enum as a public field, so you can use it in the template. To have type safety

File: libs/dh/wholesale/feature-calculations/src/lib/create/create.component.html [hunk 2]
```diff
@@ -21,162 +21,111 @@
...
-            ) {
-              <watt-field-error>
-                {{ t("create.period.invalid") }}
-              </watt-field-error>
-            }
-          </watt-datepicker>
+      <!-- Process type -->
+      <watt-dropdown
+        [label]="t('create.calculationType.label')"
+        [formControl]="formGroup.controls.calculationType"
```
Comment (mimse) [inline libs/dh/wholesale/feature-calculations/src/lib/create/create.component.html:83]: Praise: Nice refactor to formControl

**Review by mimse — CHANGES_REQUESTED** [review]
Good work, just a few comments

File: libs/dh/wholesale/feature-calculations/src/lib/create/create.component.html (hunk 1 above)
Comment (ManBearTM) [reply libs/dh/wholesale/feature-calculations/src/lib/create/create.component.html]: Good idea!
37008b8fc41d90453fbff87a6bf18607f67c9f07

File: libs/dh/wholesale/feature-calculations/src/lib/create/create.component.html [hunk 3]
```diff
@@ -21,162 +21,112 @@
...
+          type="danger"
+          icon="danger"
+          message="{{
+            t('create.quarterlyResolutionTransitionError', {
+              resolutionTransitionDate: resolutionTransitionDate | wattDate
+            })
+          }}"
+        />
+      } @else if (
+        latestPeriodEnd && formGroup.controls.calculationType.value === "BALANCE_FIXING"
```
Comment (mimse) [inline libs/dh/wholesale/feature-calculations/src/lib/create/create.component.html:49]: Suggestion: Do the same here :-)

Comment (ManBearTM) [reply libs/dh/wholesale/feature-calculations/src/lib/create/create.component.html]: b0a7711c7f0e94b92570696dac1126837411c45f

**Review by mimse — APPROVED** [review]

**Pull request merged**